matchy.py

```diff
@@ -106,6 +131,6 @@ def create_group_chat(users: [str]):
     save_partners(new_pairs, previous_partners)
 
     # Create a groupchat on Slack for each pair
-    for pair in new_pairs:
-        create_group_chat(pair)
-
+    # for pair in new_pairs:
+        # create_group_chat(pair)
+    create_group_chat(new_pairs)    # for testing
```

Comment:
Usually you want to avoid committing code changes from testing, but we can update this in the next commit